src/client/pages/Debugger/components/QuoteData.tsx

```diff
@@ -325,63 +332,72 @@ export const QuoteData: React.FC<OfferProps> = ({ sessionToken }) => {
                 {(props) => (
                   <>
                     <Form>
-                      <InputField
-                        label="First name"
-                        placeholder=""
-                        {...props.getFieldProps('firstName')}
-                      />
-                      <InputField
-                        label="Last name"
-                        placeholder=""
-                        {...props.getFieldProps('lastName')}
-                      />
-                      <InputField
-                        label="Birth date"
-                        placeholder="2012-12-12"
-                        {...props.getFieldProps('birthDate')}
-                      />
-                      {currentMarket === Market.Se && (
-                        <InputField
-                          label="ssn"
-                          placeholder=""
-                          {...props.getFieldProps('ssn')}
-                        />
-                      )}
-                      <InputField
-                        label="Start Date (optional)"
-                        placeholder="2020-03-13"
-                        {...props.getFieldProps('startDate')}
-                      />
-                      <InputField
-                        label="Email"
-                        placeholder=""
-                        {...props.getFieldProps('email')}
-                      />
+                      {!props.isSubmitting && (
```

Comment:
What is happening here? Are we hiding the input fields while we are submitting? If so, what's the rationale behind that?

Reply:
Yes. It comes from a suggestions made by Ellen [here](https://github.com/HedvigInsurance/web-onboarding/pull/659#discussion_r742668805).

> Since you can submit the form by selecting an input field and pressing return key it makes sense that a loading indicator would be visible even if you don't scroll to the bottom of the page

Reply:
Hmm okey I see the point 👍